unreleased.txt

```diff
@@ -1,11 +1,3 @@
-All unreleased changes of klarna-payments-for-woocommerce are documented in this file.
-
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-
-## [Unreleased]
-
-------------------
-## [4.1.3] - 2025-05-23
-### Changed
-* Added support for placing orders without an address when address information isn't required, such as for virtual products.
\ No newline at end of file
+== Unreleased ==
+= 2025-05-23     - version 4.1.3 =
```

Comment:
We should not have version numbers here or dates, that will be set when we make the release, just lines for each change.